.github/workflows/ci-short.yml

```diff
@@ -154,17 +154,19 @@ jobs:
             -DMACHINE_VARIANT=hip-mpi \
             -DCMAKE_CXX_COMPILER=hipcc
       # Test example with "variables" and output
-      - name: advection
-        run: |
-          cmake --build build -t advection-example
-          cd build
-          ctest -R regression_mpi_test:output_hdf5
+      #- name: advection
+      #  run: |
+      #    cmake --build build -t advection-example
+      #    cd build
+      #    ctest -R regression_mpi_test:output_hdf5
       # Test example with swarms
       - name: particle-leapfrog
         run: |
           cmake --build build -t particle-leapfrog
           cd build
-          ctest -R regression_mpi_test:particle_leapfrog
+          /usr/bin/mpiexec -n 2 --allow-run-as-root /__w/parthenon/parthenon/build/example/particle_leapfrog/particle-leapfrog -i /__w/parthenon/parthenon/tst/regression/test_suites/particle_leapfrog/parthinput.particle_leapfrog parthenon/job/problem_id=gold --kokkos-map-device-id-by=mpi_rank
```

Comment:
Does this fix the behavior that allowed the CI to pass even when this failed?

Reply:
This was actually me doing experiments trying to catch why the CI was still failing (the `sort()` issue); I reverted this change.